app/Espo/Services/ScheduledJob.php

```diff
@@ -95,6 +97,36 @@ protected function isScheduledValid(Entity $entity): bool
         return true;
     }
 
+    public function executeNow(string $id)
+    {
+        $entity = $this->getRepository()->get($id);
+        if (empty($entity)) {
+            throw new NotFound();
+        }
+
+        $date = date("Y-m-d H:i:59");
+        $result = $this->getRepository()->getConnection()->createQueryBuilder()
+            ->select(['id'])
+            ->from('job')
+            ->where("deleted = 0 and scheduled_job_id='$id' and TIMESTAMPDIFF(MINUTE , execute_time, '$date')=0")
```

Comment:
rewrite it please.

->where('deleted =: deleted')->setParameter('deleted', false)
->andWhere('scheduled_job_id = :scheduledJobId')->setParameter('scheduledJobId', $id)
...

Reply:
and try to do not use spesific  functions, because the query should work with posgres too.